mmv1/products/notebooks/api.yaml

```diff
@@ -462,6 +466,480 @@ objects:
+                description: |
+                  The Compute Engine tags to add to runtime (see [Tagging instances]
+                  (https://cloud.google.com/compute/docs/
+                  label-or-tag-resources#tags)).
+                item_type: Api::Type::String
+              - !ruby/object:Api::Type::KeyValuePairs
+                name: "guestAttributes"
+                description: |
+                  The Compute Engine guest attributes. (see [Project and instance
+                  guest attributes](https://cloud.google.com/compute/docs/
+                  storing-retrieving-metadata#guest_attributes)).
+                output: true
+              - !ruby/object:Api::Type::KeyValuePairs
+                name: "metadata"
+                description: |
+                  The Compute Engine metadata entries to add to virtual machine.
+                  (see [Project and instance metadata](https://cloud.google.com
+                  /compute/docs/storing-retrieving-metadata#project_and_instance
+                  _metadata)).
+              - !ruby/object:Api::Type::KeyValuePairs
+                name: "labels"
+                description: |
+                  The labels to associate with this runtime. Label **keys** must
+                  contain 1 to 63 characters, and must conform to [RFC 1035]
+                  (https://www.ietf.org/rfc/rfc1035.txt). Label **values** may be
+                  empty, but, if present, must contain 1 to 63 characters, and must
+                  conform to [RFC 1035](https://www.ietf.org/rfc/rfc1035.txt). No
+                  more than 32 labels can be associated with a cluster.
+              - !ruby/object:Api::Type::Enum
+                name: "nicType"
+                description: |
+                  The type of vNIC to be used on this interface. This may be gVNIC
+                  or VirtioNet.
+                values:
+                  - UNSPECIFIED_NIC_TYPE
+                  - VIRTIO_NET
+                  - GVNIC
+      - !ruby/object:Api::Type::Enum
+        name: 'state'
+        values:
+          - STATE_UNSPECIFIED
+          - STARTING
+          - PROVISIONING
+          - ACTIVE
+          - STOPPING
+          - STOPPED
+          - DELETING
+          - UPGRADING
+          - INITIALIZING
+        description: |
+            The state of this runtime.
+        output: true
+      - !ruby/object:Api::Type::Enum
+        name: 'healthState'
+        values:
+          - HEALTH_STATE_UNSPECIFIED
+          - HEALTHY
+          - UNHEALTHY
+        description: |
+            The health state of this runtime.
+        output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'accessConfig'
+        description: |
+          The config settings for accessing runtime.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: 'accessType'
+            values:
+              - RUNTIME_ACCESS_TYPE_UNSPECIFIED
+              - SINGLE_USER
```

Comment:
recommend string with link to documentation
https://cloud.google.com/vertex-ai/docs/workbench/reference/rest/v1/projects.locations.runtimes#RuntimeAccessType

Reply:
Done